src/components/app-top-bar.js

```diff
@@ -170,7 +175,13 @@ const CustomSuffixRenderer = ({ props, element }) => {
     );
 };
 
-const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
+const AppTopBar = ({
+    user,
+    tabIndex,
+    onChangeTab,
+    userManager,
+    setIsComputationRunning,
```

Comment:
Instead of going back all the way to the top with this and pass it down only to use it in a SLD, could we move this status in the redux ?

Reply:
Done